src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -1928,9 +1928,13 @@ NOTE: if you're sure you want to do this, please open an issue as to why. In the
 
         cmd.arg("--json");
 
+        if builder.config.rust_debug_assertions {
```

Comment:
nit: can we rename this to rust_debug_assertions_rustc? Or maybe just rustc_debug_assertions and std_debug_assertions for the other one?

Reply:
Yes, we can rename them to look less similar in prefix.